doc/examples/tls.rst

```diff
@@ -118,13 +118,19 @@ PyMongo can be configured to present a client certificate using the
   ...                              tls=True,
   ...                              tlsCertificateKeyFile='/path/to/client.pem')
 
-If the private key for the client certificate is stored in a separate file use
-the ``ssl_keyfile`` option::
+If the private key for the client certificate is stored in a separate file,
+it should be concatenated with the certificate file. For example, to
+concatenate PEM-formatted certificate file ``cert.pem`` and keyfile ``key.pem``
+into a single file ``concat_client.pem``, on *nix systems users can run::
+
+  $ cat key.pem cert.pem > concat_client.pem
+
+PyMongo can be configured with the concatenated certificate keyfile using the
+``tlsCertificateKeyFile`` option::
 
   >>> client = pymongo.MongoClient('example.com',
   ...                              tls=True,
-  ...                              tlsCertificateKeyFile='/path/to/client.pem',
-  ...                              ssl_keyfile='/path/to/key.pem')
+  ...                              tlsCertificateKeyFile='/path/to/concat_client.pem')
 
 Python supports providing a password or passphrase to decrypt encrypted
 private keys. Use the ``tlsCertificateKeyFilePassword`` option::
```

Comment:
Hmmm, what's the upgrade path for users of this example? Can we remove `ssl_keyfile` here?

Reply:
I am not sure I follow? Users will need to concat the files...